src/Umbraco.Web/Unversion/UnversionComponent.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Umbraco.Core;
+using Umbraco.Core.Composing;
+using Umbraco.Core.Logging;
+using Umbraco.Core.Services;
+using Umbraco.Web.Scheduling;
+
+namespace Umbraco.Web.Unversion
+{
+    public class UnversionComponent : IComponent
+    {
+        private readonly IRuntimeState _runtime;
+        private readonly IProfilingLogger _logger;
+        private readonly IContentService _contentService;
+        private readonly IUnversionService _unversionService;
+        private readonly BackgroundTaskRunner<IBackgroundTask> _unversionRunner;
+
+        public UnversionComponent(IRuntimeState runtime, IProfilingLogger logger, IContentService contentService, IUnversionService unversionService)
+        {
+            _runtime = runtime;
+            _logger = logger;            
+            _contentService = contentService;
+            _unversionService = unversionService;
+            _unversionRunner = new BackgroundTaskRunner<IBackgroundTask>("Unversion", _logger);
+        }
+
+        public void Initialize()
+        {
+            int delayBeforeWeStart = 60000; // 60000ms = 1min
+            int howOftenWeRepeat = 6000 * 60; // 60000 * 60 = 1hr
```

Comment:
In the PR description, you say that the unversioning happens once every 24 hours, but here it happens every hour.

Reply:
Thanks @skttl yes its supposed to be every hour AFAIK. Will update the description of PR 👍